### PR TITLE
MM-13545 Removal of unused status flags in Schemes requests state object

### PR DESCRIPTION
**Pull request by nadalfederer**
Remove unused status flags from https://github.com/mattermost/mattermost-redux/blob/master/src/reducers/requests/schemes.js

Fixes https://github.com/mattermost/mattermost-server/issues/13029

PR's for reference:
Redux PR: mattermost/mattermost-redux#748
Mobile PR: mattermost/mattermost-mobile#2490
Web PR: mattermost/mattermost-mobile#2496

### CODE REVIEW

File: src/actions/schemes.ts [hunk 1]
```diff
@@ -50,24 +44,15 @@ export function createScheme(scheme: Scheme): ActionFunc {
...
-            data: null,
-        }, getState);
         let data = null;
         try {
             data = await Client4.deleteScheme(schemeId);
         } catch (error) {
             forceLogoutIfNecessary(error, dispatch, getState);
-            dispatch(batchActions([
-                {type: SchemeTypes.DELETE_SCHEME_FAILURE, error},
-                logError(error),
```
Comment (sudheerDev) [inline src/actions/schemes.ts]: @nadalfederer hey, Can you add the dispatch of logError back here? We still would like to keep those.

**Comment by sudheerDev** [comment]
@nadalfederer everything looks good to me other than the one change request.

**Comment by nadalfederer** [comment]
@sudheerDev thanks for the review. I fixed it. Off-topic: there is a `getSchemeChannels` in schemes.ts. I've not found its using in any other files/projects. Isn't it redundant?

**Comment by jasonblais** [comment]
@sudheerDev quick reminder to help with reviews after the holidays, thanks! :)

**Comment by sudheerDev** [comment]
> @sudheerDev thanks for the review. I fixed it. Off-topic: there is a `getSchemeChannels` in schemes.ts. I've not found its using in any other files/projects. Isn't it redundant?

Thanks ok. It is fine to leave. We don't need to remove actions which are not used with this PR.  

@nadalfederer Hey, there are few conflicts. Can you resolve them so i can merge it?. Sorry about the delay, Somehow this was out of my review queue until jason requested again.

**Comment by nadalfederer** [comment]
@sudheerDev hi, I resolved conflicts

**Pull request merged**